rllib/algorithms/algorithm.py

```diff
@@ -1588,46 +1596,69 @@ def add_policy(
                 returns False) will not be updated.
             evaluation_workers: Whether to add the new policy also
                 to the evaluation WorkerSet.
-            workers: A list of RolloutWorker/ActorHandles (remote
+            worker_list: A list of RolloutWorker/ActorHandles (remote
```

Comment:
I have a random request, do you mind staying with ``workers`` for now? since this is a simple name change.
as part of the elastic training PR, I am getting rid of all these places where we are accessing underlying RolloutWorkers outside of WorkerSet.
so if we deprecate this today, in a few days, I am gonna have to deprecate worker_list too, and we will have 2 deprecated fields here.